charms/istio-pilot/src/charm.py

```diff
@@ -115,10 +116,8 @@ def __init__(self, *args):
         # * relation_joined: because we send data to the other side whenever we see a related app
         # * relation_changed: because of SDI's data versioning model, which first agrees on the
         #                     schema version and then sends the rest of the data
-        # TODO: * relation_broken: is this needed?
         self.framework.observe(self.on["istio-pilot"].relation_created, self.reconcile)
         self.framework.observe(self.on["istio-pilot"].relation_changed, self.reconcile)
-        self.framework.observe(self.on["istio-pilot"].relation_broken, self.reconcile)
```

Comment:
rel_changed is not called on broken. Are we OK with it?

Reply:
Yeah, this relation is entirely an outward transmission of data.  We send out info about the istio daemon to whoever listens, and don't care if they stop listening.